py/picca/delta_extraction/expected_fluxes/true_continuum.py

```diff
@@ -351,6 +351,34 @@ def read_raw_statistics(self):
         except:
             raise ExpectedFluxError(f"raw statistics file {filename} couldn't be loaded")
 
+        header = hdul[1].header
+        if Forest.wave_solution == "log":
+            if (
+                header['LINEAR'] 
+                or not np.isclose(header['L_MIN'], 10**Forest.log_lambda_min, rtol=1e-3) 
+                or not np.isclose(header['L_MAX'], 10**Forest.log_lambda_max, rtol=1e-3)  
+                or not np.isclose(header['LR_MIN'], 10**Forest.log_lambda_min_rest_frame, rtol=1e-3)
+                or not np.isclose(header['LR_MAX'], 10**Forest.log_lambda_max_rest_frame, rtol=1e-3)
+                or not np.isclose(header['DEL_LL'], Forest.delta_log_lambda, rtol=1e-3)
+            ):
+                raise ExpectedFluxError(f'''raw statistics file pixelization scheme does not match input pixelization scheme. 
+                \t\tL_MIN\tL_MAX\tLR_MIN\tLR_MAX\tDEL_LL
+                raw\t{header['L_MIN']}\t{header['L_MAX']}\t{header['LR_MIN']}\t{header['LR_MAX']}\t{header['DEL_LL']}
+                input\t{10**Forest.log_lambda_min}\t{10**Forest.log_lambda_max}\t{10**Forest.log_lambda_min_rest_frame}\t{10**Forest.log_lambda_max_rest_frame}\t{Forest.delta_log_lambda}''')
+        elif Forest.wave_solution == "lin":
+            if (
+                not header['LINEAR'] 
+                or not np.isclose(header['L_MIN'], Forest.lambda_min , rtol=1e-3)
+                or not np.isclose(header['L_MAX'], Forest.lambda_max , rtol=1e-3)
+                or not np.isclose(header['LR_MIN'], Forest.lambda_min_rest_frame, rtol=1e-3)
+                or not np.isclose(header['LR_MAX'], Forest.lambda_max_rest_frame, rtol=1e-3)
+                or not np.isclose(header['DEL_L'], Forest.delta_lambda, rtol=1e-3)
+            ):
+                raise ExpectedFluxError(f'''raw statistics file pixelization scheme does not match input pixelization scheme. 
+                \tL_MIN\tL_MAX\tLR_MIN\tLR_MAX\tDEL_LL
+                raw\t{header['L_MIN']}\t{header['L_MAX']}\t{header['LR_MIN']}\t{header['LR_MAX']}\t{header['DEL_LL']}
+                input\t{Forest.lambda_min}\t{Forest.lambda_max}\t{Forest.lambda_min_rest_frame}\t{Forest.lambda_max_rest_frame}\t{Forest.delta_lambda}''')
```

Comment:
Thanks for adding this consistency test. Have you checked that it works, when you run it with the wrong setting?

Reply:
What do you mean by it works? I thought we wanted it to fail if different pixelization schemes were given for raw_stats_file and input

Reply:
I would add a test using the wrong settings and expecting the ExpectedFluxError